patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -49,7 +49,7 @@
  
  		public static void EnterWorld(int playerIndex)
  		{
-+			Logging.Terraria.InfoFormat("Entering world with player: {0}, IsCloud={1}, Size={2}x{3}", Main.ActivePlayerFileData.Name, Main.ActivePlayerFileData.IsCloudSave, Main.maxTilesX, Main.maxTilesY);
++			Logging.Terraria.InfoFormat("Entering world with player: {0}, IsCloud={1}, Size={2}x{3}, Evil: {4}, IsExpert: {5}", Main.ActivePlayerFileData.Name, Main.ActivePlayerFileData.IsCloudSave, Main.maxTilesX, Main.maxTilesY, WorldGen.crimson.ToInt(), Main.expertMode);
```

Comment:
Why log `expertMode` and not `master` or `journey`? Consider `GameMode` instead?